ref/general/requestTiming.adoc

```diff
@@ -0,0 +1,71 @@
+:page-layout: general-reference
+:page-type: general
+:page-description: The request timing feature automatically detects slow requests and logs the information when the request is completed.
+:page-categories: MicroProfile
+:seo-title: The request timing feature automatically detects slow and hung requests
+:seo-description: The request timing feature automatically detects slow and hung requests and logs the information when the request is completed.
+= Slow and hung request detection
+
+Slow or hung requests can happen, and your application needs a continuous monitoring tool to detect the cause.
+The request timing feature automatically detects slow  and hung requests and logs the information when the request is completed.
+
+The `requestTiming` feature monitors requests with respect to time and automatically detects and diagnoses slow and hung requests.
```

Comment:
use plain text name. If you have to use the monospace/code name, include the version number- but avoid that if possible